KarhooUISDK/Screens/SideMenu/SideMenuContent/MenuContentPresenter.swift

```diff
@@ -11,8 +11,8 @@ import KarhooSDK
 final class MenuContentScreenPresenter: MenuContentPresenter {
 
     private let routing: SideMenuHandler
-    private let mailConstructor: KarhooFeedbackEmailComposer
-    private weak var view: MenuContentView?
```

Comment:
Why was the weak specifier removed?

Reply:
On the unit tests, the menu content view was being garbage collected thus being evaluated as null. Are there any considerations for the view having a weak reference ?

Reply:
Done